python_modules/dagster/dagster/_core/definitions/decorators/asset_check_decorator.py

```diff
@@ -61,25 +61,25 @@ def asset_check(
         compute_kind (Optional[str]): A string to represent the kind of computation that executes
             the check, e.g. "dbt" or "spark".
         retry_policy (Optional[RetryPolicy]): The retry policy for the op that executes the check.
-        severity (AssetCheckSeverity): Severity of the check.
+        severity (AssetCheckSeverity): Severity of the check. Defaults to WARN.
 
     Produces an :py:class:`AssetChecksDefinition` object.
 
     Examples:
         .. code-block:: python
```

Comment:
For each example, we should have a line explaining what the code block is doing. Otherwise, in the rendered API docs, it will be confusing to the reader about what this is showcasing.

Reply:
added a line to the second one